processing/src/main/java/org/apache/druid/query/context/ResponseContext.java

```diff
@@ -84,36 +125,81 @@
     ETAG("ETag"),
     /**
      * Query fail time (current time + timeout).
+     * The final value in comparison to continuously updated TIMEOUT_AT.
```

Comment:
I failed to understand this sentence after several readings. Please reword.

Reply:
reworded it